winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+Here, the columns having FirstName and LastName as MappingName are excluded when exporting.
+
+### Excel version
+
+While exporting to Excel, specify the excel version by using the `ExcelVersion` property.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting stacked headers
+
+Export the stacked headers to excel by setting the `CanExportStackedHeaders` property to `true`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.CanExportStackedHeaders = true;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting merged cells
+
+Export the merged cells to excel by setting the `CanExportMergedCells` property as `true`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.CanExportMergedCells = true;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Export column width to Excel
+
+You can export the columns with its actual width by setting the `CanExportColumnWidth` property to `true`. 
+
+{% tabs %}
+{% highlight c# %}
+var options =new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.CanExportColumnWidth = true;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+The default value of the `CanExportColumnWidth` property is `true`.
+
+### Export with customized row height
+
+You can export the TreeGrid to Excel with customized row height by using the `DefaultRowHeight` property.
+
+{% tabs %}
+{% highlight c# %}
+var options =new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.DefaultRowHeight = 60;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Change the node expand state in Excel
+
+You can change the node expanding state in Excel by using the `NodeExpandMode` property.

```

Comment:
Change the node expanding state in Excel by using the `NodeExpandMode` property.

Reply:
updated